README.md

```diff
@@ -284,8 +271,67 @@ from TwitchChannelPointsMiner import TwitchChannelPointsMiner
 twitch_miner = TwitchChannelPointsMiner("your-twitch-username")
 twitch_miner.mine(followers=True, blacklist=["user1", "user2"])  # Blacklist example
 ```
+
+### By cloning the repository
+1. Clone this repository `git clone https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2`
+2. Install all the requirements `pip install -r requirements.txt` . If you have problems with requirements, make sure to have at least Python3.6. You could also try to create a _virtualenv_ and then install all the requirements
+```sh
+pip install virtualenv
+virtualenv -p python3 venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
 Start mining! `python run.py` 🥳
 
+### pip
+Install the package via pip, you will find a stable version - maybe a different version from the master branch.
+- `pip install Twitch-Channel-Points-Miner-v2`
+- Exceute the run.py file `python run.py` 🥳
+
+### Docker
+
+The following file is mounted :
+
+- run.py : this is your starter script with your configuration
+
+These folders are mounted :
+
+- analytics : to save the analytics data
+- cookies : to provide login information
+- logs : to keep logs outside of container
+
+**Example using docker-compose:**
+
+```yml
+version: "3.9"
+
+services:
+  miner:
+    image: tkdalex/twitch-channel-points-miner-v2
+    tty: true
+    environment:
+      - TERM=xterm-256color
+    volumes:
+      - ./analytics:/usr/src/app/analytics
+      - ./cookies:/usr/src/app/cookies
+      - ./logs:/usr/src/app/logs
+      - ./run.py:/usr/src/app/run.py:ro
+    ports:
+      - "5000:5000"
+```
+
+Example with docker run:
+```sh
+docker run \
+    -v $(pwd)/analytics:/usr/src/app/analytics \
```

Comment:
Not sure $(pwd) would work on windows. It's probably safer to use ./

Reply:
damn windows :rage:

Reply:
On Linux the `./` doesn't work `docker: Error response from daemon: create ./analytics: "./analytics" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.` :rofl:

Reply:
You should probably use mounts in the docker run: `--mount type=bind,source=./analytics,target=/usr/src/app/analytics`

Reply:
As info, for windows, $(pwd) will work with powershell but not with cmd.exe.